mapbox/svg-to-geojson
# Add option to accept id attributes from <path>

### The PR
Currently, the code ignores any id attributes. This PR handles id attributes on `<path>` and returns the id as a feature property.  It also comments out the svgo plugin `cleanupIDs` because it stripped attributes altogether. 

### Reasoning 
The reasoning behind this change is that we want to be able to edit the SVGs in  Mapbox Studio and possibly do some data-join at runtime. Without a specific property, this is hard to accomplish. 

We could possibly handle multiple attributes on `<path>` but for now, this will make it easier to do data-driven styling using the svg in Studio. 

### To test

Use the `WSJ.svg` from this link: http://bl.ocks.org/veltman/c7bfb3d4a3817f7ee0bf2dd19ff058c1. 

Upload it, and download it. Check to make sure the GeoJSON has an `id` property. If no `id` is passed, the properties will be empty.
